### DISCUSSION TITLE
Moving Unrelated Repositories

### DISCUSSION THREAD

**Discussion by Corionis** [GrokSoft/ELS#135] [General 1. Discussions]
Preparing to re-brand this project as "**_Corionis ELS_**". To that end my other unrelated projects are being moved to [Monkeyboy-Com](https://github.com/Monkeyboy-Com).

Why? Because Googling "ELS" returns many other results. However, "**_Corionis ELS_**" will be unique, once it's indexed by Google and other search engines. All this is in preparation for up-coming releases, new sub-projects, and various promotions (_awareness_) efforts, for ELS - all under the Corionis account.

Eventually I will be seeking people to take-over Corionis, ELS and all that is involved. Including a dedicated email account and associated Google Analytics account. 

Contributors willing to jump-in are encouraged. I could use some help. It has a lot of potential.

**Comment by Monkeyboy-Com** [comment]
Confirm move of unrelated projects to  [Monkeyboy-Com](https://github.com/Monkeyboy-Com). Some still need to be moved and modified to accommodate the changes.
